test/views/DeclarationDetailsViewSpec.scala

```diff
@@ -557,6 +557,30 @@ class DeclarationDetailsViewSpec extends UnitViewSpec with GivenWhenThen with In
       }
     }
 
+    def createSubmissionWithRejectedAmendment(notificationSummaries: Seq[NotificationSummary]) = {
```

Comment:
Maybe you could avoid to create this new method. You already have `createSubmissionWith`, isn't it? You could you use it just modifying its signature...
```
def createSubmissionWith(notificationSummaries: Seq[NotificationSummary], requestType: RequestType = SubmissionRequest)
```